docs-ref-services/ai-formrecognizer-readme-pre.md

```diff
@@ -3,15 +3,15 @@ title: Azure Form Recognizer client library for Java
 keywords: Azure, java, SDK, API, azure-ai-formrecognizer, formrecognizer
 author: maggiepint
 ms.author: magpint
-ms.date: 07/07/2020
+ms.date: 08/12/2020
```

Comment:
This should be 08/11 ?
@scbedd

Reply:
I don't think we'll invest to handle the UTC vs local timezone here, but will assist with a manual patch if we want to fix it!